src/content/docs/journey-large-logs/organize.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Organize your logs 
+metaDescription: "Test"
+---
+
+import logsParsing from 'images/logs_screenshot_full-parsing.webp'
+import logsPartition from 'images/logs_screenshot_full-partition.webp'
+
+Once you've decided what logs to ingest and store, it's time to organize your logs. You're still likely ingesting hundreds of gigabytes or dozens of terabytes of logs. While it's a lot less than you originally had, you'll still have toil while trying to effectively use them. 
+
+New Relic provides the ability to group these remaining logs into thematic partitions and parse the logs to pull out and tag valuable attributes. By organizing your logs in this manner, you can:
```

Comment:
`New Relic provides the ability to`

let's try to find a way to remove this, there's not much semantic content to these words and the rest of your setnence is (a) full of user value and (b) already qutie complex.